adb/adb_protocol.py

```diff
@@ -19,10 +19,10 @@
 
 import struct
 import time
-
+from io import BytesIO
+from future.utils import iteritems
```

Comment:
http://python-future.org/reference.html?highlight=iteritems#future.utils.iteritems

Use viewitems?

Reply:
Jut reverted to items below. Now there is no reliance on the future library